src/sgame/sg_cmds.cpp

```diff
@@ -1572,6 +1575,13 @@ static bool isDisabledVoteType(const char *vote)
 	return false;
 }
 
+static bool botFillVoteArgValid( char *arg )
```

Comment:
This should use the same parsing function as BotFillCmd. Which is currently `atoi`, but it would be better to change them both to Str::ParseInt to forbid junk after the number.

Reply:
I changed this to using Str::ParseInt, but did not change BotFillCmd yet.

Reply:
Well, i changed BotFillCmd too. `atoi` is evil.